Userland/Applications/DisplaySettings/BackgroundSettings.gml

```diff
@@ -39,7 +39,7 @@
 
             @GUI::Button {
                 name: "wallpaper_open_button"
-                tooltip: "Select wallpaper from file system."
+                tooltip: "Select wallpaper from file system"
```

Comment:
The commit message here refers to "the button" but you are really changing a *tooltip* and a FilePicker title.

Reply:
@awesomekling done! changed "the button" to "tooltip and window title" (couldn't really find a better name that would fit in the char limit, but I think it's good enough).

Is there anything else that needs to be improved?